app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<h2>Log in</h2>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %><br />
+    <%= f.password_field :password, autocomplete: "current-password" %>
+  </div>
+
+  <% if devise_mapping.rememberable? %>
+    <div class="field">
+      <%= f.check_box :remember_me %>
+      <%= f.label :remember_me %>
+    </div>
+  <% end %>
+
+  <div class="actions">
+    <%= f.submit "Log in" %>
+  </div>
+<% end %>
+
+<%= render "devise/shared/links" %>
```

Comment:
- [ ] Great job on developing a functional web app! However, it appears that you may have overlooked one of the requirements, which is to ensure that your design matches the provided wireframe. Currently, your web app is utilizing the default styling of the browser, which deviates from the intended design. I recommend updating your design to align with the wireframe specifications.
- Your design:
  ![firefox_Hus3efygXm](https://github.com/tjay1760/Blog-App/assets/104968236/629cbf84-604c-4ab1-bac0-6ca0842602ed)
- Wireframe design:
  ![firefox_F87DeENYem](https://github.com/tjay1760/Blog-App/assets/104968236/a45b716a-0af9-4248-bc67-166f84502ba1)